.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: recursive
-          fetch-depth: 0
+          fetch-depth: 1
```

Comment:
This build's checkout phase took 2m13s vs [36s](https://github.com/goatcorp/Dalamud/actions/runs/5259586413/jobs/9505352763) for a prior PR. Fluke?

Reply:
Seems so - this only took 26s on my fork.